From be634872b696db33427f1bca4e8f833658055b30 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Tue, 23 May 2017 20:05:28 +0800
Subject: [PATCH 1804/5242] MLK-14967-1 ARM64: dts: freescale: imx8qm: reserve
 DDR space for M4

commit  5c2b12eebaa6fa5da80a87402f5e99dafa112739 from
https://source.codeaurora.org/external/imx/linux-imx.git

Reserve first 4M memory for below usage:

0x80000000 - 0x800fffff: ARM Trusted Firmware;
0x80100000 - 0x803fffff: M4/RPMSG;

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts  |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
index 20f9929..480391f 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
@@ -16,7 +16,8 @@
 /dts-v1/;
 
 /* First 128KB is for PSCI ATF. */
-/memreserve/ 0x80000000 0x00020000;
+/* Last 3M is for M4/RPMSG */
+/memreserve/ 0x80000000 0x00400000;
 
 #include "fsl-imx8qm.dtsi"
 
-- 
1.7.9.5

